### ISSUE TITLE
Cannot use loadTable with a p5.file

### ISSUE THREAD

**Issue by Falkonn91** [processing/p5.js#3005]
#### Nature of issue?

- [x] Found a bug
- [ ] Existing feature enhancement
- [ ] New feature request

#### Most appropriate sub-area of p5.js?

- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] Events
- [ ] Image
- [x] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Other (specify if possible)

#### Which platform were you using when you encountered this?

- [ ] Mobile/Tablet (touch devices)
- [x] Desktop/Laptop
- [ ] Others (specify if possible)

#### Details about the bug: 

- p5.js version: 0.6.1
- Web browser and version: Firefox v60.0.2
- Operating System: Windows 7 64 bits
- Steps to reproduce this:

Hello ! I can't use loadTable() with a p5.file, I don't know if it's really a bug or if it's just totally impossible :/
In my project, I have to open a tsv table selected by the user with createInputFile(), the problem is when I try to use the function loadTable like that: `loadTable(file, "tsv")` it return the error "TypeError: path.lastIndexOf is not a function" at line 59796 of p5.js

**Comment by limzykenneth** [comment]
You can't use the file uploaded with `createFileInput()` with `loadTable()` as `loadTable()` is used to load from a server location, eg with a URL link, so the first argument passed to `loadTable()` needs to be a URL string. For your use case, we may need to create a new `createTable()` function that can take in csv ot tsv strings to initialize a new `p5.Table` object.

**Comment by Falkonn91** [comment]
Ha ok, thank you for your answer ^^

**Comment by Trimad** [comment]
This is still a problem 😐

**Comment by ricardocanada** [comment]
I too would like to be able to load (or create) a table from createFileInput() . Has this project been assigned to anyone yet?

**Comment by limzykenneth** [comment]
@ricardocanada We are actually considering removing p5.Table support in the upcoming major version of p5.js but would like to know your experience with it and your use case to help us make a decision.

**Comment by ricardocanada** [comment]
I only recently tried table commands as a database solution. They seemed to do the job better than other methods I have used. But I suppose I could get similar results with JSON if you discontinue table.Richard BourneCalgary, Alberta, Canada On Aug 13, 2024, at 3:27 AM, Kenneth Lim ***@***.***> wrote:﻿
@ricardocanada We are actually considering removing p5.Table support in the upcoming major version of p5.js but would like to know your experience with it and your use case to help us make a decision.
